source/community/release_notes/redwood.rst

```diff
@@ -25,10 +66,140 @@ Instructor Experiences
 Administrators & Operators
 **************************
 
+Settings and Toggles
+====================
+
+Waffle flags:
```

Comment:
are these all new flags?

Reply:
It's a fine question. I'm going to throw out the current list and use the one @mariajgrimaldi generated. I don't trust my work. I was trying to guess and that's NEVER what you want.